aws/resource_aws_waf_web_acl_test.go

```diff
@@ -231,9 +234,10 @@ func TestAccAWSWafWebAcl_Rules(t *testing.T) {
 	resourceName := "aws_waf_web_acl.test"
 
 	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSWaf(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckAWSWafWebAclDestroy,
+		PreCheck:            func() { testAccPreCheck(t); testAccPreCheckAWSWaf(t) },
+		Providers:           testAccProviders,
+		CheckDestroy:        testAccCheckAWSWafWebAclDestroy,
+		DisableBinaryDriver: true,
```

Comment:
We should double check this one -- Its not obvious from looking at the `TestStep` why it would have failed.

Reply:
Near the end of looking through over 400 errors I started to lose focus, it's possible I marked the odd one wrong

Reply:
Great eye! This was a mistake, fixing now